ql/src/test/org/apache/hadoop/hive/ql/parse/repl/metric/TestReplicationMetricCollector.java

```diff
@@ -60,6 +60,7 @@
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
```

Comment:
Please remove the wildcart import from the static import.

Reply:
Hi, thank you for spotting this. I fixed this and the other import-only changes.